src/jvmMain/kotlin/io/beatmaps/api/search.kt

```diff
@@ -107,6 +108,8 @@ class SearchApi {
 
 fun <T> Op<Boolean>.notNull(b: T?, block: (T) -> Op<Boolean>) = if (b == null) this else this.and(block(b))
 
+fun Op.Companion.of(b: Boolean): Op<Boolean> = if (b) Op.TRUE else Op.FALSE
```

Comment:
I slightly prefer the way it is now, but I find it hard to express why. I'll try though.
Having the function on `Op` will isolate it to code that's already using Ops. Having it on `Boolean` will make it available for every boolean, while most of the time it would go unused.